addons/html_editor/static/src/main/powerbox/powerbox_plugin.js

```diff
@@ -125,7 +125,7 @@ export class PowerboxPlugin extends Plugin {
         power_buttons: withSequence(100, {
             commandId: "openPowerbox",
             description: _t("More options"),
-            icon: "fa-ellipsis-v",
+            icon: "oi oi-ellipsis-v",
```

Comment:
I see you added the `oi` class here but it seems the `fa` class is not removed.
We should keep only the right class depending to the icon used :) 
![Capture d’écran 2025-04-07 à 11 08 10](https://github.com/user-attachments/assets/7e0ac046-a1ac-417f-a183-29b06b04c7ee)